GOVERNANCE.md

```diff
@@ -42,9 +45,40 @@ Maintainers have right to nominate contributors to become maintainers and vote o
 
```

Comment:
> Maintainers are eligible to vote on Technical Committee elections. They can be nominated as members to the Technical Committee during Technical Committee elections.

--> All maintainers, regardless of which Eiffel project they are maintainer of, are eligible...